packages/rps/src/redux/message-service/saga.ts

```diff
@@ -48,7 +48,7 @@ interface WalletMessage {
 type Message = AppMessage | WalletMessage;
 
 export function* sendWalletMessageSaga() {
-  const sendMessageChannel = createWalletEventChannel([Wallet.MESSAGE_RELAY_REQUESTED]);
+  const sendMessageChannel = yield call(createWalletEventChannel, [Wallet.MESSAGE_RELAY_REQUESTED]);
```

Comment:
Any specific reason to convert this function invocation to the `yield call`? Typically, `yield call` is employed to make side effects unit-testable...

Reply:
I was following the tutorial in the redux-saga docs, and their examples do it this way. I had also read that not `yield`-ing effects inside a `while` loop can cause the app to hang (that's not an issue here but that may have influenced me to use the `yield call` here). 

 Looking in to it further however, I think you're right that it only seems to serve to make testing easier. Reverting this for now.